engine/gamesys/src/gamesys/resources/res_sprite.cpp

```diff
@@ -114,10 +114,12 @@ namespace dmGameSystem
                 dmResource::PreloadHint(params->m_HintInfo, ddf->m_Textures[i].m_Texture);
             }
         }
-        else
+
+        if (ddf->m_TileSet[0] != '\0')
         {
-            dmResource::PreloadHint(params->m_HintInfo, ddf->m_TileSet);
+            dmLogInfo("Using tilesets for sprites is deprecated. '%s' will not be loaded or used.", ddf->m_TileSet);
```

Comment:
so we can't use tilesets in sprites at all? isn't it a breaking change?

Reply:
Hmm, that's not great if that is the case.

Reply:
It is not used anywhere in the code. It is not a breaking change, we don’t even release it?

Reply:
It is deprecated in the proto file in favor of the texture/sampler field.

Reply:
![CleanShot 2024-08-05 at 14 03 33@2x](https://github.com/user-attachments/assets/db9fd860-f1c9-4399-96eb-b11842009e2f)

Ah, it's not related to the ability to set a tilesource as a texture on a sprite?

Reply:
No, that should end up as an entry in the textures field. The tile_source field is deprecated in the source format and should not be used at all in the engine

Reply:
Thank you for this explanation!